moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -333,23 +335,23 @@ const ProjectView = () => {
   };
 
   /**
-   * Finds the status_id for a phase name
+   * Finds the status_id for a status name
    * @param {string} phase - The name of the phase
    * @returns {number}
    */
-  const resolveStatusIdForPhase = phase =>
-    data?.moped_status.find(s => s.status_name.toLowerCase() === phase)
+  const resolveStatusIdForStatusName = status =>
+    data?.moped_status.find(s => s.status_name.toLowerCase() === status)
       .status_id ?? 1;
 
   /**
-   * Updates status of the current project
+   * Updates status of the current project to either on-hold or canceled
    */
-  const handleUpdateStatus = current_phase => {
+  const handleUpdateStatus = new_status => {
```

Comment:
I'm changing the variable name here because in this case, we only ever call this function with either "on-hold" or "canceled" which are statuses and project phases.

Reply:
This is triggered when someone cancels or puts the project on hold via the drop down to the right of the project title.